Assets/Scripts/Battle/BattleSystem/BattleController.cs

```diff
@@ -12,7 +12,7 @@ public class BattleController : MonoBehaviour
     private List<BattleParticipant> enemies;
 
     private List<BattleParticipant> battleParticipants = new List<BattleParticipant>();
-    public List<BattleParticipant> aliveEnemies { get { return enemies.FindAll(participant => !participant.Dead); } }
+    public List<BattleParticipant> aliveEnemies { get { return enemies.FindAll(participant => !participant.IsDead); } }
```

Comment:
Looks like the rename back to `Dead` wasn't added to the latest commit. Make sure this is renamed back to `Dead` everywhere.